payment-component/vanilla-js/apps/client/scripts/checkout.js

```diff
@@ -0,0 +1,71 @@
+async function getData() {
+    const url = "http://localhost:8082/payment-intent";
+    try {
+        const response = await fetch(url, {
+            method: "POST",
+        });
+        if (!response.ok) {
+            throw new Error(`Response status: ${response.status}`);
+        }
+        const json = await response.json();
+        console.log(json.id);
+        const chargebee = window.Chargebee.init({
+            site: "hp-internal-us-test",
+            publishableKey: "test_w9wYSMChRqylCiz2bacqUgYgxIFnVfZZ",
+        })
+        const components = chargebee.components({});
+        const onSuccess = (payment_intent) => {
+            console.log(payment_intent);
+        }
+        const onError = (error) => {
+            // handle payment errors here
+            console.log(error);
+        }
+        const onPaymentMethodChange = (error) => {
+            // handle payment errors here
+            console.log(error);
+        }
+        const paymentComponentOptions = {
+            paymentIntentId: json.id,
+            layout: {
+                type: 'tab',
+                showRadioButtons: true,
+            },
+            paymentMethods: {
+                sortOrder: ["paypal_express_checkout", "card"],
+                allowed: ["paypal_express_checkout", "card", "google_pay"]
+            },
+            locale: "fr",
```

Comment:
As discussed, can we set locale and other global parameters in the `components()` call?